### DISCUSSION TITLE
Create a connector to access open data SODA API (Socrata now Tyler Technologies)

### DISCUSSION THREAD

**Discussion by mamontesp** [airbytehq/airbyte#36104] [New Connector Request]
## Tell us about the new connector you’d like to have
Connector to use public data exposed through SODA APIs by public governments. 
 
* Which source and which destination?
Source: Endpoints with format https://www.datos.gov.co/resource/ceth-n4bn.json
Destination: Postgres 

* Do you need a specific version of the underlying data source e.g: you specifically need support for an older version of the API or DB? 
No

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data? This helps us understand the use case.
I want to create some full search engines using the public data provided by Colombian government.

* How often do you want to run syncs?
Every 24 hours 
* If this is an API source connector, which entities/endpoints do you need supported?
Yes, it is an API source connector. I'll need to support all the endpoints available in the page www.datos.gov.co

* If the connector is for a paid service, can we name you as a mutual user when we subscribe for an account? Which company should we name? 
No, the data is publicly available 

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?
Use the file connector, but it does have some pagination issues. 

## Are you willing to submit a PR?
Yes

**Comment by mamontesp** [comment]
I'll be working on this :)

**Comment by natalyjazzviolin** [comment]
@mamontesp looking forward to your PR! :)

[a comment by an automated account is not shown]
